services/app-api/serverless.yml

```diff
@@ -36,6 +36,9 @@ custom:
   tableArn: ${env:AMENDMENTS_TABLE_ARN, cf:database-${self:custom.stage}.AmendmentsTableArn}
   atomicCounterTableName: ${env:AMENDMENTS_COUNTER_TABLE_NAME, cf:database-${self:custom.stage}.AmendmentsAtomicCounterTableName}
   atomicCounterTableArn: ${env:AMENDMENTS_COUNTER_TABLE_ARN, cf:database-${self:custom.stage}.AmendmentsAtomicCounterTableArn}
+  associateWaf:
+    name: ${self:service.name}-${self:custom.stage}-WebACLAssociation
+    version: V2
```

Comment:
Per the plugin docs, this should be set under 'custom', not the top level.  I'm not totally sure this was working before hand, but it's working for sure now.